### PR TITLE
Reader: Add in feed and site state in redux state tree

### PR DESCRIPTION
**Pull request by blowery**
Things left to do:

- [x] Don't refetch feeds that we already have
- [ ] Refetch feeds that we do have that are "stale". Stale is > 24 hours old.
- [ ] Prune the serialized state tree? Maybe?

### CODE REVIEW

**Comment by blowery** [comment]
cc @bluefuton @TooTallNate

**Comment by bluefuton** [comment]
Looks good thus far 👍

**Comment by blowery** [comment]
bailing on  the last two for now
- Refetch feeds that we do have that are "stale". Stale is > 24 hours old.
- Prune the serialized state tree? Maybe?

**Comment by bluefuton** [comment]
All looks good to :ship: as far as I can see :)

File: client/state/reader/sites/schema.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { sitesSchema } from 'state/sites/schema';
+
+export const readerSitesSchema = Object.assign( {}, sitesSchema );
+
+readerSitesSchema.patternProperties['^\\d+$'].properties.feed_ID = { type: 'number' };
```
Comment (mcsf) [inline client/state/reader/sites/schema.js]: @blowery, I stumbled upon this file just now. `assign` is shallow, so I think we're mutating `sitesSchema` here, right?

screen shot 2016-06-25 at 17 32 52

Comment (blowery) [inline client/state/reader/sites/schema.js]: yup, quite right!

Comment (blowery) [inline client/state/reader/sites/schema.js]: https://github.com/Automattic/wp-calypso/pull/6409 should fix it @mcsf

**Pull request merged**